.env.example

```diff
@@ -15,6 +15,11 @@ HEADLESS="true"
 # at the same brokerage with a comma, then separate account credentials with a colon
 # BROKER=BROKER_USERNAME:BROKER_PASSWORD,OTHER_BROKER_USERNAME:OTHER_BROKER_PASSWORD
 
+# BBAE Pro
+# USE_EMAIL takes either TRUE or FALSE. TRUE mean email for username, FALSE means phone number
+# BBAE=BBAE_USERNAME:BBAE_PASSWORD:USE_EMAIL TRUE/FALSE
+BBAE=
```

Comment:
I still think having a TRUE/FALSE field will confuse users. This could easily be handled by having even a rudimentary check, like an "@" character means email.